# Vedder VESC open source motor controller state (telemetry)

float64 rotor_pos
